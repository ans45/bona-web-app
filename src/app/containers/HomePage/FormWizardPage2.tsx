import Box from '@material-ui/core/Box';
import { FormWizardPage } from 'app/components/FormWizard/FormWizardPage';
import { FormWizardPageTitle } from 'app/components/FormWizardPageTitle';
import React from 'react';
import { Field } from 'react-final-form';

export const FormWizardPage2 = () => (
  <FormWizardPage validate={() => undefined} submitButtonText="Submit">
    <Box mb={4}>
      <FormWizardPageTitle>
        Question one is lorem ipsum dolor ipsum?
      </FormWizardPageTitle>
      <Field
        name="userInput"
        component="input"
        type="text"
        label="User Input"
      />
    </Box>
  </FormWizardPage>
);
