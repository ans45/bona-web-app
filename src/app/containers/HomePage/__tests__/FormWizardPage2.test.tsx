import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPage2 } from '../FormWizardPage2';
import { Form } from 'react-final-form';

describe('<FormWizardPage2  />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Form onSubmit={() => undefined}>{() => <FormWizardPage2 />}</Form>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
