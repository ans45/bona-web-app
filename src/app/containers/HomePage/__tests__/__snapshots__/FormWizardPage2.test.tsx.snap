// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPage2  /> should match snapshot 1`] = `
<div
  class="MuiBox-root MuiBox-root-1"
>
  <div
    class="MuiBox-root MuiBox-root-4"
  >
    <h2
      class="MuiTypography-root makeStyles-root-2 makeStyles-root-3 MuiTypography-h5"
    >
      Question one is lorem ipsum dolor ipsum?
    </h2>
  </div>
  <div
    class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
  >
    <label
      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink="false"
    >
      User Input
    </label>
    <div
      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
    >
      <input
        aria-invalid="false"
        class="MuiInputBase-input MuiOutlinedInput-input"
        name="textInput"
        type="text"
        value=""
      />
      <fieldset
        aria-hidden="true"
        class="PrivateNotchedOutline-root-5 MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="PrivateNotchedOutline-legendLabelled-7"
        >
          <span>
            User Input
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
